addons/MetroidvaniaSystem/Database/MapEditor.gd

```diff
@@ -26,6 +26,11 @@ func _ready() -> void:
 	get_current_sub_editor()._editor_enter()
 	MetSys.settings.theme_changed.connect(grid.queue_redraw)
 	map_overlay.mouse_entered.connect(map_overlay.grab_focus)
+	map_overlay.set_drag_forwarding(
+		func(_at_pos): return null,
```

Comment:
```suggestion
		Callable(),
```
You can pass empty Callable to skip a method.
I think with this, the method call can be in a single line.

Reply:
Fixed.